test/hello.spec.ts

```diff
@@ -0,0 +1,41 @@
+import 'mocha';
+import { helloWorld, helloUser } from '../src/handlers/handler';
+import { StubEvent, StubContext } from '../stubs/hello';
+import expect = require('expect.js');
```

Comment:
Tiny nitpick, but syntax for keeping this consistent with the other imports would be `import * as expect from 'expect.js`.